lessons/1.4-Namespaces/text.adoc

```diff
@@ -0,0 +1,15 @@
+= Namespaces
+
+When you write a program, and use more and more names, coming up with new ones
+becomes a problem. Even more so, when you use somebody else's code, which you
+_included_ into your file. To solve this issue, C++ has _namespaces_.  Those
+are named _scopes_, that contain other named entities, like variables and
```

Comment:
`scopes` isn't really a term I've heard used; it'll only serve to confuse. Better to be clear about scope rather than using an ambiguous term.